asab/web/rest/json.py

```diff
@@ -79,7 +87,7 @@ async def JsonExceptionMiddleware(request, handler):
 	# HTTP errors to JSON
 	except aiohttp.web.HTTPError as ex:
```

Comment:
@ateska  This section just takes what's already in the HTTP headers and puts it into JSON body.
There is no other app specific information available. Only the exception message, and that's being put into the `message` JSON field.

What should be the `result` string then? I can't get more specific than what the HTTP code says here. Or should I delete this whole section?

Reply:
The `result` field should contain:
- `OK` - to indicate success & to allow simple `if`-based check
- !`OK`; in many cases `ERROR` will be just fine

Eventually, we can introduce error codes that will describe a reason for failure that could be displayed to user in a localised way. `ERROR-12345` will be the form.

Replicating the HTTP status message is creating a false structure that will result in eventual refactorization b/c ppl will use that incorrectly.

Reply:
`NOT-FOUND` message is fine mainly b/c we use that in many places already and it highlight the typical issue - making request on non-existing server-side entity.

`NOT-AUTHORIZED` is likely also fine, b/c it is descriptive.

... but similarity to HTTP status codes is more or less incidental.

Reply:
`BAD-REQUEST` & `SERVER-ERROR` is plainly `ERROR` ... there is no need to provide more info - it is in the HTTP header for those, who are interested.

Reply:
Ok, I simplified it to only three possible error `result` strings: 

`NOT-AUTHORIZED`, `NOT-FOUND` and `ERROR`